### PR TITLE
Rewrite readme (GEA-11653)

### PR DESCRIPTION
**Pull request by kenany**

### CODE REVIEW

File: pangea-sdk/v3/README.md [hunk 1]
```diff
@@ -1,60 +1,50 @@
...
+A Go SDK for integrating with Pangea services.
 
 ## Installation
 
-To add Pangea Go SDK to your project, will need to run next command on your project root directory where you should have `go.mod` file:
-
-```
-go get github.com/pangeacyber/pangea-go/pangea-sdk
+```bash
+$ go get github.com/pangeacyber/pangea-go/pangea-sdk
```
Comment (pangea-andrest) [inline pangea-sdk/v3/README.md:17]: shouldn't add `/v3`?

**Review by pangea-andrest — DISMISSED** [review]
Just a comment

File: pangea-sdk/v3/README.md (hunk 1 above)
Comment (kenany) [reply pangea-sdk/v3/README.md]: So this whole time the instructions have been pointing users to v1? 😟 

Fixed.

Comment (pangea-andrest) [reply pangea-sdk/v3/README.md]: I'm afraid so

File: pangea-sdk/v2/README.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ A Go SDK for integrating with Pangea Services.
 
 To add Pangea Go SDK to your project, will need to run next command on your project root directory where you should have `go.mod` file:
 
-```
-go get github.com/pangeacyber/pangea-go/pangea-sdk
+```bash
+$ go get github.com/pangeacyber/pangea-go/pangea-sdk/v2
```
Comment (pangea-andrest) [inline pangea-sdk/v2/README.md:27]:
```suggestion
$ go get github.com/pangeacyber/pangea-go/pangea-sdk/v3
```

**Review by pangea-andrest — CHANGES_REQUESTED** [review]

File: pangea-sdk/v2/README.md (hunk 2 above)
Comment (kenany) [reply pangea-sdk/v2/README.md:27]: This is v2's readme.

**Review by pangea-andrest — APPROVED** [review]

**Pull request merged**